src/annotator/config/test/index-test.js

```diff
@@ -95,28 +95,78 @@ describe('annotator.config.index', function () {
     );
   });
 
-  [
-    'assetRoot',
-    'branding',
-    'openSidebar',
-    'requestConfigFromFrame',
-    'services',
-  ].forEach(function (settingName) {
-    it('returns the ' + settingName + ' value from the host page', function () {
-      const settings = {
-        assetRoot: 'chrome-extension://1234/client/',
-        branding: 'BRANDING_SETTING',
-        openSidebar: 'OPEN_SIDEBAR_SETTING',
-        requestConfigFromFrame: 'https://embedder.com',
-        services: 'SERVICES_SETTING',
-      };
-      fakeSettingsFrom().hostPageSetting = function (settingName) {
-        return settings[settingName];
-      };
-
-      const settingValue = configFrom('WINDOW')[settingName];
-
-      assert.equal(settingValue, settings[settingName]);
+  ['branding', 'openSidebar', 'requestConfigFromFrame', 'services'].forEach(
+    function (settingName) {
+      it(
+        'returns the ' + settingName + ' value from the host page',
```

Comment:
This could could be modernized a little here to use string interpolation and arrow functions.